core/src/main/java/org/hisp/dhis/android/core/trackedentity/NewTrackerImporterTrackedEntityAttributeValue.java

```diff
@@ -34,38 +34,29 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 import com.gabrielittner.auto.value.cursor.ColumnAdapter;
 import com.google.auto.value.AutoValue;
 
 import org.hisp.dhis.android.core.arch.db.adapters.custom.internal.DbDateColumnAdapter;
 import org.hisp.dhis.android.core.common.CoreObject;
-import org.hisp.dhis.android.core.trackedentity.internal.TrackedEntityAttributeValueFields;
 
 import java.util.Date;
 
 @AutoValue
-@JsonDeserialize(builder = AutoValue_NewTrackerImporterTrackedEntityAttributeValue.Builder.class)
 public abstract class NewTrackerImporterTrackedEntityAttributeValue implements CoreObject {
 
     @Nullable
-    @JsonProperty(TrackedEntityAttributeValueFields.ATTRIBUTE)
     public abstract String trackedEntityAttribute();
 
     @Nullable
-    @JsonProperty()
     @JsonInclude()
     public abstract String value();
 
     @Nullable
-    @JsonProperty()
     @ColumnAdapter(DbDateColumnAdapter.class)
     public abstract Date createdAt();
 
     @Nullable
-    @JsonProperty()
     @ColumnAdapter(DbDateColumnAdapter.class)
     public abstract Date updatedAt();
 
```

Comment:
The annotation `@JsonIgnore` in `trackedEntityInstance()` can be removed. Actually, the DTO must not have this property, it should be removed